packages/block-library/src/navigation/frontend.js

```diff
@@ -14,6 +14,10 @@ function navigationToggleModal( modal ) {
 	triggerButton.setAttribute( 'aria-expanded', ! isHidden );
 	closeButton.setAttribute( 'aria-expanded', ! isHidden );
 	modal.classList.toggle( 'has-modal-open', ! isHidden );
+
+	// Add a class to indicate the modal is open.
+	const toggleClass = ( el, className ) => el.classList.toggle( className );
+	toggleClass( document.querySelector( 'html' ), 'has-modal-open' );
```

Comment:
I think it would be good to use `classList.toggle` like we did above; directly on the element. It doesn't seem to be necessary — yet — to add a new function for this.

```suggestion
	const htmlElement = document.querySelector( 'html' );
	htmlElement.classList.toggle( 'has-modal-open' );
```